clang/test/Sema/exprs.c

```diff
@@ -57,7 +57,7 @@ void test4(void) {
 
       var =+5;  // no warning when the subexpr of the unary op has no space before it.
       var =-5;
-  
```

Comment:
Handful of WS changes in this file, though not sure I care enough to have you change this.